kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/manager/ResourceManager.java

```diff
@@ -43,6 +53,9 @@
 public class ResourceManager {
```

Comment:
Maybe worth a comment on this class about what it's for and why we need this static bag of Resources, clients, and ExtensionContext

Reply:
This method will be replaced soon by the test-frame approach, so we can skip adding new comments if you agree